source/parse.h

```diff
@@ -2613,6 +2613,51 @@ struct declaration_node
     }
 
 
+    struct should_add_default_constructor_result {
```

Comment:
Minor consistency tweak: In other places these are usually named `funcname_ret` rather than `funcname_result`

Reply:
Renamed (and not yet pushed).